### PR TITLE
TW 378 Add Apple Pay and Google Pay to Invoicing section

### PR DESCRIPTION
**Pull request by HeatherStenson**
This PR:

- Adds Apple Pay and Google Pay to the [Paying by invoice](https://learning.postman.com/docs/administration/billing/#paying-by-invoice) section of the Billing page
- Reformats the [Paying by invoice](https://learning.postman.com/docs/administration/billing/#paying-by-invoice) section to make the note about only being available to Enterprise customers more prominent

### CODE REVIEW

**Comment by HeatherStenson** [comment]
@jkonrath-postman and @claireriley - Kalpana signed off on the content and wording for this PR in Slack convo, so this PR is ready to review from a docs perspective.

**Review by claireriley — APPROVED** [review]
LGTM!

File: src/pages/docs/administration/billing.md [hunk 1]
```diff
@@ -138,7 +144,7 @@ Add or update your company name, billing email, address, and your VAT ID if appl
 
 ### Failed payments
 
-If you have a card on file, Postman will automatically attempt 3 times to charge it. If this is unsuccessful, team members with billing privileges receive an email notification.
+If you have a card on file, Postman will automatically attempt 3 times to charge it. If this is unsuccessful, team members with billing privileges receive an email notification. Use the link in the email to access and pay for your [invoice](#using-the-pay-now-option).
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/billing.md:147]:
```suggestion
If you have a card on file, Postman will automatically attempt three times to charge it. If this is unsuccessful, team members with billing privileges receive an email notification. Use the link in the email to access and pay for your [invoice](#using-the-pay-now-option).
```

**Review by jkonrath-postman — APPROVED** [review]
One little thing, but LGTM!

**Pull request merged**